standup_notes/__main__.py

```diff
@@ -73,6 +73,16 @@ def read_note(note):
         print(note + ' doesn\'t exist yet.')
 
 
+def next_weekday(day: date) -> date:
+    tomorrow = day + timedelta(days=1)
+    week_num = tomorrow.weekday()
```

Comment:
This should probably be: 
```suggestion
    day_num = tomorrow.weekday()
```
`week_num` sounds like it's the number of the week in the year (e.g. 1 of 52, 2 of 52, etc.)